spec/lib/open_food_network/scope_variants_to_search_spec.rb

```diff
@@ -73,7 +73,7 @@
         v2.save!
         v3.save!
         v4.save!
-        expect(result.map(&:display_name)).to eq ["Product 1", "Product 1 - a", "Product 1 - b", "Product 1 - c"]
+        expect(result.map(&:display_name)).to eq ["Product 1 - a", "Product 1 - b", "Product 1 - c", "Product 1"]
```

Comment:
Oh, do we want this change in order? It doesn't seem intuitive to me. Is it due to [NULL values](https://www.postgresql.org/docs/8.3/queries-order.html) or depending on the product name?

Reply:
Yes, it depends on the product.name:
```
(byebug) result[3].name
"Product 4"
```
(as we set `v4.display_name = "Product 1"`)

Don't know if sorting by name is a good idea then.

Reply:
Then, I've updated the PR by commit d4af03dc5

Reply:
Hang on, we are going in circles here. So your spec is looking at `display_name` and that didn't seem intuitive with the new order including the product. But I thought that the view (or the serializer) actually prints `"#{object.name} - #{object.full_name}"` which refers to the product's name. So if you want to test the order as displayed when adding a product then you have to test for this, not just display_name.

Maybe we should add a spec for the admin variant serializer? That should be much closer to what we see on the page.

I'm sorry for pointing in the wrong direction before. I just asked a question because I didn't actually know the answer. I hadn't thought that through.

Reply:
I did a last (?) _force push_ commit. I didn't add a spec for the admin variant serializer (mainly because out of scope, but actually it could be interesting/going crazy to implement such a thing)
Hope it pass the review now ;)